[2024-07-30T10:27:39.104+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:27:39.140+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-18T00:00:00+00:00 [queued]>
[2024-07-30T10:27:39.154+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-18T00:00:00+00:00 [queued]>
[2024-07-30T10:27:39.154+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:27:39.234+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-18 00:00:00+00:00
[2024-07-30T10:27:39.244+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1205) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:27:39.246+0000] {standard_task_runner.py:63} INFO - Started process 1209 to run task
[2024-07-30T10:27:39.244+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-18T00:00:00+00:00', '--job-id', '211', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpfr8vk9ks']
[2024-07-30T10:27:39.248+0000] {standard_task_runner.py:91} INFO - Job 211: Subtask Run_data_quality_checks
[2024-07-30T10:27:39.374+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-18T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:39.756+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-18T00:00:00+00:00'
[2024-07-30T10:27:39.758+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:39.789+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:27:39.807+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:39.828+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:27:39.834+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:39.837+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 3901 records
[2024-07-30T10:27:39.838+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:27:39.855+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:39.875+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:27:39.879+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:39.882+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 86 records
[2024-07-30T10:27:39.883+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:27:39.899+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:39.986+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:27:39.992+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:39.995+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:27:39.996+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:27:40.013+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:40.042+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:27:40.048+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:40.052+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:27:40.053+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:27:40.071+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:40.094+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:27:40.100+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:40.102+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 3623 records
[2024-07-30T10:27:40.103+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:40.124+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-18T00:00:00+00:00, execution_date=20181118T000000, start_date=20240730T102739, end_date=20240730T102740
[2024-07-30T10:27:40.185+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:40.220+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:27:40.294+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
